client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -33,7 +33,7 @@ import { getDowngradePlanFromPurchase } from 'state/purchases/selectors';
 class CancelPurchaseButton extends Component {
 	static propTypes = {
 		purchase: PropTypes.object.isRequired,
-		selectedSite: PropTypes.object.isRequired,
```

Comment:
`selectedSite` is still used in the component (although not required anymore) so shouldn't it still be declared here?

Reply:
You are correct. I mistakenly deleted these when I meant to only delete the `.isRequired` portion. Sorry, I'm new to propTypes. 🙃